webpack/dev.config.js

```diff
@@ -41,21 +41,17 @@ module.exports = {
       { test:/\.json$/,
         loader: 'json-loader'
       },
-      {
-        test: /\.css$/,
-        loaders: [
-          'style',
-          'css?modules&importLoaders=1&sourceMap&localIdentName=[local]',
-          'postcss'
-        ]
-      },
       { test: /\.scss$/,
-        loaders: [
+        loader: ExtractTextPlugin.extract(
           'style',
-          'css?modules&importLoaders=2&sourceMap&localIdentName=[local]__[hash:base64:5]',
-          'postcss',
-          'sass'
-        ]
+          'css?modules&importLoaders=2&sourceMap&localIdentName=[name]__[local]___[hash:base64:5]' +
+          '!postcss' +
+          '!sass'
+        )
+      },
+      {
+        test: /\.css$/,
+        loader: ExtractTextPlugin.extract('style-loader', "css-loader")
       },
```

Comment:
Have you tested the result without changing this config file?
I think it will work normally without changing it.